services/web-app/components/divider/divider.js

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright IBM Corp. 2022, 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import clsx from 'clsx'
+import PropTypes from 'prop-types'
+
+import * as styles from './divider.module.scss'
+
```

Comment:
You'll want to add a description here that will show up in storybook

```
/**
 * The `<Divider>` component is a wrapper component that adds a top border
 * and spacing to divide sections of content.
 */
```

Or something like that